package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/rpx-xui-common-lib",
-  "version": "0.2.5",
+  "version": "1.0.1",
```

Comment:
Still needs to be a pre-release version, not an actual version. (Actual version isn't used until we do a proper release)